packages/reaction-i18n/private/data/i18n/en.json

```diff
@@ -523,6 +523,7 @@
           "passwordRequired": "Password is required.",
           "passwordMustBeAtLeast6CharactersLong": "Password must be at least 6 characters long.",
           "passwordMustContainRequirements": "Password must contain at least one number or symbol.",
+          "usernameMinLength": "Username must be at least 3 characters long",
           "minChar": "6 character minimum password.",
           "pwdsDontMatch": "Passwords don't match",
           "pwOneDigit": "Password must have at least one digit.",
```

Comment:
(not your code but..) Don't these message contradict each other? Is it one number or symbol or one digit? Also we should probably rephrase "6 character minimum password" as "Password much contain at least 6 characters"